### ISSUE TITLE
Moving from Mail Poet to Campaign Monitor for EdTech Demos

### ISSUE THREAD

**Issue by murphybarb** [BCcampus/bcc-sage#247]
## Submit a feature request or bug report

- [X] This is a feature request
- [ ] This is a bug report
- [ ] This request isn't a duplicate of an [existing issue](https://github.com/BCcampus/bcc-sage/issues)
- [ ] You cleared your Web Browsers cache to ensure all new styling and images were loaded. 

---

**What is the current behavior?**

**Investiage** what is required, and estimate the time, to move from using Mail Poet to Campaign monitor on the new BCcampus.ca for the EdTech sign up and notifcation process. 

Reference: Users signs up for EdTech demo notifications [using form here on EdTech](https://edtech.bccampus.ca/edtech-projects/edtech-demo/)

Functions of the Mail Poet plugin on EdTech:  
- users use reg form to sign up for the notification subscrption list 
- Clint sends email notifications to the subscription list informing of upcoming Edtech demos
 
Current Registration form fields are: 
- First name (required information)
- Last name (required information)
- Primary institution (pull down menu) - lists primary institutions + "other", 26 total) 
- email (required information)

Form Button to submit information 
- "Subscribe!" button 

**What is the expected or desired behavior?**

Duplicate the sign up and notfication process on the new BCcampus.ca, but using Campaign Monitor instead of mail poet. Duplicate the form entry fields and ability to use a pull down menu to choose "Primary Institution".

Current Registration form fields are: 
- First name (required information)
- Last name (required information)
- Primary institution (pull down menu) - lists primary institutions + "other", 26 total) 
- email (required information)

Form Button to submit information 
- "Subscribe!" button 

The sign up form will live on BCcampus.ca on a page on it's own, not embedded within the content, like it currenlty is on EdTech. There will be a link **"Sign up for EdTech Demonstrations"** from [this page on the new BCcampus.ca](https://bccampus.ca/ed-tech/edtech-demos/) to the new sign up form. 

The EdTech subscription sign up form would be intended to duplicate the look and feel of the [BCcampus newsletter subscription form for the new BCcampus.ca](https://helga.bccampus.ca/wp/bcc/subscribe/?subscribe=
)   

---

## Feature Request

**Please provide use cases for changing the current behavior:**

EdTech content is consolidating into the new BCcampus.ca. 

**Other relevant information:**

We are favouring moving the EdTech subscription list to the Campaign Monitor platform, but also have submitted a ticket (#245 ) to investigate the time and what is required to stay with using the Mail Poet plugin. Once the investigation is complete we will decide on which option and submit a ticket to complete the work.

**Comment by bdolor** [comment]
Steps: 
1. Mailpoet export list from edtech with appropriate fields
2. Clean/merge data from list to prepare it for import into Campaign Monitor
3. Create new edtech list in campaign monitor
4. Import/add users with appropriate fields.
5. Create custom fields and signup form in campaign monitor
6. Create new subscription button component in bcc-sage theme
7. Create new page template in bcc-sage theme for custom campaign monitor form
8. Quality assurance that components and form are in the expected places 
9. Quality assurance that the workflow and sign up form work as expected. 

Assumption is that the newsletter 'template' that goes out is out of scope/covered by mark/comm

Initial assumption is that this is a 'Big' task...or approx 7 hours.

**Comment by murphybarb** [comment]
@bdolor 

I want to confirm an item regarding the building of the form in campaign monitor. First I need to figure out how the form is imported/embedded from campaign monitor into a page in wordpress. I'm using the page that Alex built for the newsletter subscription as a guide. I can't determine how the form was imported/embedded on this page from campaign monitor. https://helga.bccampus.ca/wp/bcc/subscribe/

There is not short code or embedd code when you "edit page". Not sure how that form appears on the page. 

Are you able to assit?

**Comment by murphybarb** [comment]
**1.** **Quality assurance that components and form are in the expected places**

**Requested change:** modifications required to form fields on [subscription form](https://helga.bccampus.ca/wp/bcc/bccampus/topics-of-practice/3-3-educational-technology/699190-2/edtech-demo-subscribe/) page. Update form field labels to match the form on the current [EdTech site ](https://edtech.bccampus.ca/edtech-projects/edtech-demo/). 
- 1st form field "First name"
- 2nd form field "Last name"
- 3rd form field "Email"
- "Primary Institution" Pull down menu containing list of institutions 



**2. **Quality assurance that the workflow and sign up form work as expected****

- Remove the "subscribe to EdTEch Demos" subscription box in the right-hand side bar on ["3.3 Educational Technology"](https://helga.bccampus.ca/wp/bcc/bccampus/topics-of-practice/3-3-educational-technology/). This is the incorrect page to subscribe for EdTech Demos, also do not want to treat the subscribe button the same as the newsletter subscription button, which is the purple box in the right-hand sidebar
- a subscription botton has been inserted into the content on the correct page ["3.3.4 EdTech Demos"](https://helga.bccampus.ca/wp/bcc/bccampus/topics-of-practice/3-3-educational-technology/699190-2/). I coudln't get the link to work and the styling of the button is wonky as a result. Can you fix this? The button on this page links to ["3.3.4.3 Sign up for EdTech Demonstrations"](https://helga.bccampus.ca/wp/bcc/bccampus/topics-of-practice/3-3-educational-technology/699190-2/edtech-demo-subscribe/)

**3. **Change request to the EdTech Demo subscription page****
On the [EdTech Demo subscription page ](https://helga.bccampus.ca/wp/bcc/bccampus/topics-of-practice/3-3-educational-technology/699190-2/edtech-demo-subscribe/) – remove the "Related News" and "Related Events" sections at the bottom of the page. The EdTech Demo subscription page and [BCcampus newsletter subscription pag](https://helga.bccampus.ca/wp/bcc/subscribe/?subscribe=)e should be the same and not contain "related" info at the bottom. 

**4. Confirmed, sign up form is capturing Data in Campaign Monitor**
Tested the form several times. New subscribers are being added to the newly created "EdTech" subscription list in Campaign Monitor 

**5. Subscription list name change in Campaign Monitor**
Will changning the subscription list name in campaign monitor from "EdTech" to "EdTech Demos" impact anything? We are able to change the name ourselves if this is possible without breaking anything.

**Comment by bdolor** [comment]
1. discussion occurred in standup
2. a. complete b. complete 
3. complete
4. great
5. nope
6. discussed in stand up

**Comment by murphybarb** [comment]
>     1. discussion occurred in standup -
Requested change to form not required. There are no issues with the current version of the form, which combines the first and last name form fields. 
>     2. a. complete b. complete
Looks great!
>     3. complete
Looks great!
>     4. great
Cool beans
>     5. nope
I've updated the name of the subscription list in campaign monitor to "EdTech Demos" 
>     6. discussed in stand up
To confrim, applying the "EdTech Subscribe Landing Template" to the new page on the on the live site is all that is required for the subscrition form to work?
